libs/remote_storage/src/s3_bucket.rs

```diff
@@ -158,12 +154,24 @@ impl S3Bucket {
             }
             prefix
         });
+
+        let rps = aws_config.concurrency_limit.get();
+        let concurrency_limiter = leaky_bucket::RateLimiter::builder()
+            .max(rps)
+            .initial(0)
+            // refill it by rps every second. this means the (rps+1)th request will have to wait for
+            // 1 second from earliest.
+            .refill(rps)
+            .interval(std::time::Duration::from_secs(1))
+            .fair(true)
```

Comment:
NIT: this is the default value, do you want to emphasize this?

Reply:
Yes, just blast all of the settings into our git history because they are so few. (Note the crate was not yet 1.0 when I started thinking about this :))